redirect/src/test/java/com/adyen/checkout/redirect/DefaultRedirectHandlerTest.kt

```diff
@@ -19,12 +20,14 @@ import org.robolectric.annotation.Config
 
 @RunWith(RobolectricTestRunner::class)
 @Config(manifest = Config.NONE)
-class RedirectUtilTest {
+class DefaultRedirectHandlerTest {
```

Comment:
This has nothing to do with your PR, but I hate Robolectric tests. What if we use Java's URL, can we then have a normal unit test?

Reply:
Unfortunately java `URL` is not the best class out there 😅